src/sentry/sentry_metrics/indexer/cloudspanner/cloudspanner.py

```diff
@@ -81,7 +75,7 @@ def decode(self, value: EncodedId) -> DecodedId:
         return reverse_bits(value + 2**63, 64)
 
 
-class RawCloudSpannerIndexer(StringIndexer):
+class RawCloudSpannerIndexer:
```

Comment:
I'm wondering, what's our plan for removing this? Are we going to remove this now in this same PR?

Reply:
I don't have a preference, if you don't I will create a separate pr

Reply:
The cloud spanner pr has been merged